pyupgrade.py

```diff
@@ -335,6 +382,7 @@ def _process_dict_comp(tokens, start, arg):
 
 
 def _process_is_literal(tokens, i, compare):
+    # type: (List[Token], int, ast.cmpop) -> None
```

Comment:
should this be `Union[ast.Is, ast.IsNot]`?

Reply:
yes